_posts/2021-08-31-release-1.14.0-rc0.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "Help us stabilize Apache Flink 1.14.0 RC0"
+date:   2021-08-31 00:00:00
+categories: news
+authors:
+- joemoe:
+  name: "Johannes Moser"
+
+---
+
+Dear Flink Community,
+
+we are excited to announce the first release candidate of Apache Flink 1.14. 🎉
+
+A lot of features and fixes went into this release, including improvements to the 
+unified batch and streaming experience, an increase in fault tolerance by reducing 
+in-flight data, and more developments on connectors and components. 
+It wouldn't have been possible without your help. 
+[Around 211 people](https://github.com/apache/flink/graphs/contributors?from=2021-03-31&to=2021-08-31&type=c)
```

Comment:
I'd suggest to remove this link, as it confuses people with an inconsistent number.